### PR TITLE
Feature/broad aln

### PR DESCRIPTION
**Pull request by jemten**
### This PR adds | fixes:

- Use the same bwa mem options as used by broad

### How to test:

- Automatic and continuous test suite

### Expected outcome:
- [ ] Installation, unit and integration test suite pass

### Review:
- [ ] Code review
- [ ] Tests pass

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes. If applicable record manual test results in PR header
- [ ] **MINOR** - when you add functionality in a backwards compatible manner. If applicable record manual test results in PR header
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: lib/MIP/Program/Bwa.pm [hunk 1]
```diff
@@ -27,7 +27,7 @@ BEGIN {
     our @EXPORT_OK = qw{ bwa_index bwa_mem bwa_mem2_mem bwa_mem2_index run_bwamem };
 
 }
-
+Readonly my $BASES_TO_PROCESS    => q{100000000};
```
Comment (henrikstranneheim) [inline lib/MIP/Program/Bwa.pm:30]:
```suggestion
Readonly my $BASES_TO_PROCESS    => 100_000_000;
```

Comment (henrikstranneheim) [reply lib/MIP/Program/Bwa.pm:30]: You can use underscore to separate thousands

File: t/bwa_mem.t [hunk 2]
```diff
@@ -47,6 +44,8 @@ diag(   q{Test bwa_mem from Bwa.pm}
       . $SPACE
       . $EXECUTABLE_NAME );
 
+Readonly my $BASES_TO_PROCESS => q{100000000};
```
Comment (henrikstranneheim) [inline t/bwa_mem.t:47]:
```suggestion
Readonly my $BASES_TO_PROCESS => 100_000_000;
```

File: t/bwa_mem2_mem.t [hunk 3]
```diff
@@ -47,6 +44,8 @@ diag(   q{Test bwa_mem2_mem from Bwa.pm}
       . $SPACE
       . $EXECUTABLE_NAME );
 
+Readonly my $BASES_TO_PROCESS => q{100000000};
```
Comment (henrikstranneheim) [inline t/bwa_mem2_mem.t:47]:
```suggestion
Readonly my $BASES_TO_PROCESS => 100_000_000;
```

**Review by henrikstranneheim — APPROVED** [review]
⭐ !

File: lib/MIP/Program/Bwa.pm (hunk 1 above)
Comment (jemten) [reply lib/MIP/Program/Bwa.pm:30]: Smart! I'll update the PR

**Comment by jemten** [comment]
This PR was mainly to try out the options used by broad. They don't use bwakit and I recon it's because they only do variant calling on the primary assembly. 
What do you think of the option to softclipping supplementary alignments `-Y`?

**Comment by henrikstranneheim** [comment]
> This PR was mainly to try out the options used by broad. They don't use bwakit and I recon it's because they only do variant calling on the primary assembly.
> What do you think of the option to softclipping supplementary alignments `-Y`?

I remember reading about it, but it was some time ago. I do not know - maybe it could be better for SVs? Perhaps we should test with a NIST sample?

**Pull request merged**